tests/conftest.py

```diff
@@ -0,0 +1,87 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import imp
+import parser
+import os
+import sys
+
+from lambda_local import context
+import pytest
+
+
+CONFIG_MODULE = """
+import os
+
+host = os.environ['RABBITMQ_HOST']
+port = int(os.environ['RABBITMQ_PORT'])
+user = os.environ['RABBITMQ_USER']
+password = os.environ['RABBITMQ_PASS']
+virtual_host = os.environ['RABBITMQ_VHOST']
+queue = os.environ['RABBITMQ_QUEUE']
+"""
+
+
+def build_config_module():
+    """Generate a config module and insert it
+
+    pigeon requires a config.py file which has the configuration in it. We need
+    one for tests, but that's kind of irritating. So we fake a module and stick
+    it in sys.modules.
+    """
+    st = parser.suite(CONFIG_MODULE)
+    code = st.compile('config.py')
+    cfg = imp.new_module('config')
+    exec(code, cfg.__dict__)
+    return cfg
```

Comment:
I did this because then it means we don't have to have a `config.py` file on the file system that causes confusion. We don't have to fiddle with the PYTHONPATH to make sure tests pick up the right config file and lambda picks up the right config file. Plus I got to spend 15 minutes looking into how to "import" a module the contents of which are defined in a string--that was fun.